ethminer/main.cpp

```diff
@@ -254,6 +254,9 @@ class MinerCLI
 
         app.add_option("--farm-retries", m_PoolSettings.connectionMaxRetries, "", true)->check(CLI::Range(0, 99999));
 
+        app.add_option("--retry-delay", m_PoolSettings.delayBeforeRetry, "", true)
```

Comment:
Need to integrate help text about usage of this CLI argument

Reply:
Specify unit. Seconds ? Minutes ?

Reply:
solved, see line 992